src/_gettsim/transfers/arbeitsl_geld_2/kindergelduebertrag.py

```diff
@@ -0,0 +1,113 @@
+"""Module for the calculation of the Kindergeldübertrag."""
+
+import numpy as np
+
+from _gettsim.shared import join_numpy, policy_info
+
+aggregate_by_p_id_kindergeldübertrag = {
+    "kindergeld_übertrag_m": {
+        "p_id_to_aggregate_by": "p_id_kindergeld_empf",
+        "source_col": "_kindergeld_kindbedarf_differenz_m",
+        "aggr": "sum",
+    },
+}
+
+
+def _kindergeld_per_child_m(
+    kindergeld_m: float,
+    kindergeld_anz_ansprüche: int,
+) -> float:
+    """Kindergeld per child.
+
+    Returns the average Kindergeld per child. If there are no children, the function
+    returns 0. Helper function for `kindergeld_zur_bedarfdeckung_m`.
+
+    Parameters
+    ----------
+    kindergeld_m
+        See :func:`kindergeld_m`.
+    kindergeld_anz_ansprüche
+        See :func:`kindergeld_anz_ansprüche`.
+
+    Returns
+    -------
+
+    """
+    if kindergeld_anz_ansprüche == 0:
```

Comment:
Can we ever end up here? Why?

Put differently, this seems like doubling up what is already happening via `join_numpy` below?

Reply:
Yes because first `_mean_kindergeld_per_child_m` is being calculated (for any individual, so also for those without any children) and then the whole column is used by the `join_numpy`.
That's also what the tests say when removing that condition.

Reply:
Cool, then maybe just switch the condition to go for `> 0` first, makes it easier to read (first what you care about, else other stuff)